atomicapp/requirements.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from atomicapp.constants import REQUIREMENT_FUNCTIONS
+from atomicapp.plugin import Plugin
+
+plugin = Plugin()
+plugin.load_plugins()
+
+logger = logging.getLogger(__name__)
+
+
+class Requirements:
+
+    """
+    Requirements will search what is currently being used under
+    the requirements section of Nulecule and deploy it in according
+    to the graph variables as well as whether or not said requirement
+    exists within the provider.
+
+    The variable REQUIREMENTS_FUNCTIONS lists all current function names
```

Comment:
should be "the requirements_functions dictionary maps all current requirement names to the function names for each provider"